Bottom: 077b55694d09ea9242469b56fc4bbfb43fda75c7
Top:    c9f21b31f552f54ad4c4bfc7e0a15ad98206978e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:00:36 +0000

Refresh of add-polygon-parentage-tree

---

diff --git a/src/polygon1.c b/src/polygon1.c
index b8e71f8..9089395 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -49,6 +49,7 @@
 #include "global.h"
 #include "rtree.h"
 #include "heap.h"
+#include "pcb-printf.h"
 
 #define ROUND(a) (long)((a) > 0 ? ((a) + 0.5) : ((a) - 0.5))
 
@@ -142,10 +143,9 @@ static POLYPARENTAGE no_parentage = {
   .b = NULL
 };
 
-#ifdef DEBUG
 static char *theState (VNODE * v);
 
-static void
+/* static */ void
 pline_dump (VNODE * v)
 {
   VNODE *s, *n;
@@ -160,9 +160,6 @@ pline_dump (VNODE * v)
     }
   while ((v = NEXT_VERTEX(v)) != s);
 }
-#endif
-
-static void pline_dump (VNODE * v);
 
 /*static */void
 poly_dump (POLYAREA * p)
@@ -1113,8 +1110,6 @@ cntr_in_M_POLYAREA (PLINE * poly, POLYAREA * outfst, BOOLp test)
   return FALSE;
 }				/* cntr_in_M_POLYAREA */
 
-#ifdef DEBUG
-
 static char *
 theState (VNODE * e)
 {
@@ -1139,6 +1134,7 @@ theState (VNODE * e)
     }
 }
 
+#ifdef DEBUG
 #ifdef DEBUG_ALL_LABELS
 static void
 print_labels (PLINE * a)
